src/utils/three-utils.js

```diff
@@ -357,3 +363,17 @@ export function traverseAnimationTargets(rootObject, animations, callback) {
     }
   }
 }
+
+export function createPlaneBufferGeometry(width, height, widthSegments, heightSegments, flipY = true) {
+  const geometry = new THREE.PlaneBufferGeometry(width, height, widthSegments, heightSegments);
+  // Three.js seems to assume texture flipY is true for all its built in geometry
```

Comment:
Might not be worth dealing with right now, but this comment reminds me of the motivation for turning off flipY in HubsTextureLoader in the first place... Since we are now actually not using ImageBitmapLaoder in Firefox in the first place, maybe we should just get rid of this?

Reply:
From the discussion on Discord...

Yeah, passing `imageOrientation: 'flipY'` to `ImageBitmapLoader(createImageBitmap)` may be one of the options. It's a good clean up. But I'm on the fence about it because I want to make the change minimal for the tests so far. I don't really add a change that there might be a chance to break some stuffs. (I guess it's rare tho.) We may think of it in follow up PRs?